modules/ppcp-googlepay/resources/css/styles.scss

```diff
@@ -17,3 +17,9 @@
 #ppc-button-ppcp-googlepay {
 	display: none;
 }
+
+label[for='payment_method_ppcp-googlepay'] {
+	img {
+		max-height: 25px;
+	}
+}
```

Comment:
We could consider using a plugin-wide CSS rule for all payment icons, instead of using different CSS rules for each gateway.

**Suggestion**

Undo this change, and instead modify `hosted-fields.scss` and change the first rule to:
```scss
#payment ul.payment_methods [class*=" payment_method_ppcp-"] label img {
  max-height: 25px;
  display: inline-block;
}
```